include/power_grid_model/math_solver/iterative_linear_se_solver.hpp

```diff
@@ -674,7 +704,7 @@ class IterativeLinearSESolver {
                 // get data idx of y bus,
                 // skip for a fill-in
                 Idx const data_idx = y_bus.map_lu_y_bus()[data_idx_lu];
-                if (data_idx == -1) {
+                if (data_idx == DISCONNECTED) {
```

Comment:
This does not make sense. `-1` here means different thing.

Reply:
It's then a good reason to put the contant definition inside `MeasuredValues`. So it is scoped and will not be misused.